sorc/ncep_post.fd/MISCLN.f

```diff
@@ -2221,7 +2221,7 @@ SUBROUTINE MISCLN
 !$omp parallel do private(i,j)
                DO J=JSTA,JEND
                  DO I=1,IM
-                   IF(T1D(I,J) < spval) GRID1(I,J) = EGRID2(I,J)
+                   IF(TBND(I,J,1) < spval) GRID1(I,J) = EGRID2(I,J)
```

Comment:
Should EGRID2 be checked?

Reply:
@JesseMeng-NOAA Would you modify per Bo's comments or not?

Reply:
grid1 is initialized with spval. so if (egrid2==spval) grid1=egrid2 still valid.

Reply:
@JesseMeng-NOAA If you think further modification is not needed, please set the comment as solved after your explanation.